homeassistant/components/cover/wink.py

```diff
@@ -40,13 +40,17 @@ def open_cover(self):
         """Open the shade."""
         self.wink.set_state(1)
 
+    def set_cover_position(self, position, **kwargs):
```

Comment:
There are other platforms that use this override, but won't the base class will fail:
```python
def async_set_cover_position(self, **kwargs):
        """Move the cover to a specific position.
        This method must be run in the event loop and returns a coroutine.
        """
        return self.hass.loop.run_in_executor(
            None, ft.partial(self.set_cover_position, **kwargs))
```
Could you cover this with a test?